File: src/main/java/net/softbell/bsh/dto/request/IotActionItemDto.java
package net.softbell.bsh.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 액션 아이템 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class IotActionItemDto
{
	private long actionItemId;
	private long itemId;
    private short itemStatus;
}

File: src/main/kotlin/net/softbell/bsh/dto/request/IotActionItemDto.kt
package net.softbell.bsh.dto.request

/**
 * @author : Bell(bell@softbell.net)
 * @description : 액션 아이템 DTO
 */
data class IotActionItemDto (
    var actionItemId: Long = -1,
    var itemId: Long = -1,
    var itemStatus: Double = -1.0
)